mm2src/common/common.rs

```diff
@@ -1121,9 +1121,10 @@ pub fn http_uri_to_ws_address(uri: http::Uri) -> String {
     };
 
     let host_address = uri.host().expect("Host can't be empty.");
+    let path = if uri.path() == "/" { "" } else { uri.path() };
```

Comment:
is a trailing slash problematic or just for empty path? Because we don't trim this case: `path/to/something/` here, right?

Reply:
By default uri path is "/" and we don't want that to be included in results. for `path/to/something/` having trailing slash is totally fine.